packages/forma-36-codemod/transforms/__tests__/v4.test.js

```diff
@@ -5,6 +5,7 @@ const defineInlineTest = require('jscodeshift/dist/testUtils').defineInlineTest;
 
 describe('v4 codemods', () => {
   const tests = [
+    'v4-v4-inline-entry-card',
```

Comment:
This is the only one missing:
```suggestion
    'v4-inline-entry-card',
```